extensions/amp-analytics/0.1/vendors.js

```diff
@@ -79,5 +79,22 @@ export const ANALYTICS_CONFIG = {
     },
     'optout': '_gaUserPrefs.ioo'
   }
+  
+  'atinternet': {
```

Comment:
I'd suggest sequencing this above googleanalytics above to keep things alphabetically organized.